.cicd/pipeline.yml

```diff
@@ -3,78 +3,100 @@ env:
   TEST_TIMEOUT: 60
   TIMEOUT: 120
   VERBOSE: true
+  DEBUG: true
 
 steps:
   - wait
 
   - label: ":webhook: Trigger Travis CI Build"
-    command: |
-      . ./.cicd/trigger-travis.sh
+    command:
+      - "./.cicd/travis/travis-trigger.sh"
```

Comment:
Why was the format of this command statement changed?   
Why was the script renamed?

Reply:
We've had this discussion before I believe; `command: |` is only for supporting porting over from travis and not needed for a single command.

Reply:
The script name `travis-trigger.sh` is inconsistent with the rest of the repos, which are `trigger-travis.sh`. I think we should maintain consistency in all repos.

Reply:
What about travis-build.sh? Is that not a naming convention? Why would we not change the other repos instead?

Reply:
> Is that not a naming convention?
  
Not as far as I am aware.
   
EOSIO, contracts, and eos-vm all have `trigger-travis.sh`. I don't see why we should deviate. The `.cicd` folder shouldn't be complicated.

Reply:
I changed it back to get this unblocked.